src/commands.ts

```diff
@@ -48,7 +49,12 @@ export async function enterColorHandler() {
 }
 
 export async function changeColorToRandomHandler() {
-  return await changeColor(getRandomColorHex());
+  const color = await changeColor(getRandomColorHex());
+  const message = `Peacock changed the base accent colors to ${color}, because the setting is enabled for ${
+    StandardSettings.SurpriseMeOnStartup
+  }`;
+  window.showInformationMessage(message);
+  return color;
```

Comment:
This was a fantastic idea 👍

Reply:
thanks - i felt it might be good to remind folks in case they didnt expect it.